Qualisys/QTM-Unity-Realtime-Streaming/Streaming/RTSkeleton.cs

```diff
@@ -107,9 +101,9 @@ private void BuildMecanimAvatarFromQtmTPose()
             {
                 name = this.SkeletonName,
                 position = Vector3.zero,
-                rotation = Quaternion.identity,
+                rotation = Quaternion.AngleAxis(180, Vector3.up), //TODO: Is it possible to calculate this?
                 scale = Vector3.one,
-            });
+            }); 
 
             // Create remaining T-Pose bone definitions from Qtm segments
             foreach (var segment in mQtmSkeletonCache.Segments.ToList())
```

Comment:
Is it really necessary to first convert to a list here?

Reply:
It's not